pkg/daemon/ceph/osd/daemon.go

```diff
@@ -256,15 +261,10 @@ func getDataDirs(context *clusterd.Context, kv *k8sutil.ConfigMapKVStore, desire
 	}
 
 	// the osd dirs map doesn't exist yet
```

Comment:
Should this comment also be moved? The next line below it is empty.

Reply:
this comment is still applicable, but i thought it looked a little awkward when placed directly above the next code line, because it is a general comment of where we are in the flow and its not directly related to the next code line.  Let me know if you feel strongly on this, but either way the comment is applicable and should stay.